loopchain/peer/block_manager.py

```diff
@@ -370,19 +370,15 @@ def __block_request_by_citizen(self, block_height, rs_rest_stub):
                 'height': str(block_height)
             }
         )
-        max_height_result = rs_rest_stub.call("Status")
-
-        if max_height_result.status_code != 200:
-            raise ConnectionError
-
+        last_height = rs_rest_stub.call("GetLastBlock").get('height')
```

Comment:
Deserialization is needed.
Row 'height' can be hex string.

Reply:
Resolved.